crates/autopilot/src/limit_order_quoter.rs

```diff
@@ -9,6 +9,22 @@ use model::{
 use shared::order_quoting::{OrderQuoting, QuoteParameters};
 use std::sync::Arc;
 
+#[derive(prometheus_metric_storage::MetricStorage, Clone, Debug)]
+#[metric(subsystem = "limit_order_quoter")]
+struct Metrics {
+    /// Histogram for counting ignored limit orders.
+    ignored: prometheus::Histogram,
```

Comment:
uber-naming-nit: For me, "ignored" can be confused with an order that we didn't consider at all (i.e. we decided not to quote it), and can be confused with "skipped". Maybe "failed" or something is less confusing? I don't feel super strongly though, so feel free to leave it as is.